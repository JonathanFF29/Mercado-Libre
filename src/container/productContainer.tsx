import React, { Component } from 'react';
import { Route, Switch } from 'react-router-dom';
import SearchHeader from '../template/header/search-header';
import ProductList from './productList/product-list';
import ProductDetail from './productDetail/product-detail';
import { MainDiv } from './components/mainDiv';


export class ProductContainer extends React.Component {
    
  
    render() { 
          
        return ( 
            <MainDiv >
                <Route path='/' 
                  render={(...props) => (
                      <SearchHeader />
                  )}>

                </Route>
                <Switch>
                    <Route  path="/productList" component={ProductList} /> 
                  
                    <Route   path="/productDetail" component={ProductDetail}/>
                  
                </Switch>
            </MainDiv>
        );
    }
}
 
export default ProductContainer;
