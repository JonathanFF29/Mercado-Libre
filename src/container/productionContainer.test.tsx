
import React from 'react';
import { configure, mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { ProductContainer } from './productContainer';
import renderer from 'react-test-renderer';

configure({ adapter: new Adapter() });

describe('ProductContainer', () => {
  it('ProductContainer should renders', () => {
      const wrapper = shallow(<ProductContainer/>)
      expect(wrapper).toMatchSnapshot();
  });
});