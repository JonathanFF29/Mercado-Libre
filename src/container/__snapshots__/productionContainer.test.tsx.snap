// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductContainer ProductContainer should renders 1`] = `ShallowWrapper {}`;

exports[`ProductContainer snapshot renders 1`] = `ShallowWrapper {}`;
